google-cloud-firestore/src/main/java/com/google/cloud/firestore/BulkWriterOptions.java

```diff
@@ -40,4 +56,48 @@ boolean isThrottlingEnabled() {
   public static BulkWriterOptions withThrottlingDisabled() {
     return new BulkWriterOptions(false);
   }
+
+  /**
+   * An options object that will enable throttling in the created BulkWriter with the provided
+   * starting rate.
+   *
+   * @param initialOpsPerSecond The initial maximum number of operations per second allowed by the
+   *     throttler.
+   * @return The BulkWriterOptions object.
+   */
+  @Nonnull
+  public static BulkWriterOptions withInitialOpsPerSecondThrottling(int initialOpsPerSecond) {
+    return new BulkWriterOptions(initialOpsPerSecond, DEFAULT_UNSET_VALUE);
+  }
+
+  /**
+   * An options object that will enable throttling in the created BulkWriter with the provided
+   * maximum rate.
+   *
+   * @param maxOpsPerSecond The maximum number of operations per second allowed by the throttler.
+   *     The throttler's allowed operations per second does not ramp up past the specified
+   *     operations per second.
+   * @return The BulkWriterOptions object.
+   */
+  @Nonnull
+  public static BulkWriterOptions withMaxOpsPerSecondThrottling(int maxOpsPerSecond) {
+    return new BulkWriterOptions(DEFAULT_UNSET_VALUE, maxOpsPerSecond);
+  }
+
+  /**
+   * An options object that will enable throttling in the created BulkWriter with the provided
+   * starting and maximum rate.
+   *
+   * @param initialOpsPerSecond The initial maximum number of operations per second allowed by the
+   *     throttler.
+   * @param maxOpsPerSecond The maximum number of operations per second allowed by the throttler.
+   *     The throttler's allowed operations per second does not ramp up past the specified
+   *     operations per second.
```

Comment:
Seems like this is already implied in the line above.

Reply:
N/A w/ AutoValue.